src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -458,28 +458,22 @@ Emscripten_HandleTouch(int eventType, const EmscriptenTouchEvent *touchEvent, vo
             }
             SDL_SendTouch(deviceId, id, SDL_TRUE, x, y, 1.0f);
 
-            if (!preventDefault && SDL_GetEventState(SDL_FINGERDOWN) == SDL_ENABLE) {
-                preventDefault = 1;
-            }
+            // setting preventDefault=1 breaks scrolling/pinch-to-zoom with finger events SDL_IGNOREd
+            //preventDefault = 0;
```

Comment:
This should be left alone, we still want to prevent default touch actions if the app has enabled touch events.